docs/guides/backup.md

```diff
@@ -43,23 +43,23 @@ namespace/demo created
 
 ## Overview
 
-Following diagram show how Stash takes backup of a Kubernetes volume. Open the image in new tab to see enlarged image.
+Following diagram shows how Stash takes backup of a Kubernetes volume. Open the image in new tab to see the enlarged image.
 
 <p align="center">
   <img alt="Stash Backup Flow" src="/docs/images/stash-backup.svg">
 </p>
 
-We can represent the backup flow in terms of following steps:
+The backup process consists of the following steps:
 
-1. At first, user creates a `Secret`. This secret holds the credentials to access the backend where backed up data will be stored. It also holds a password (`RESTIC_PASSWORD`) that will be used to encrypt the backed up data.
-2. Then, user creates a `Restic` crd which specifies the targeted workload for backup. It also specify the backend information where the backed up data will be stored.
-3. Stash operator watches for `Restic` crd. Once, it found a `Restic` crd, it identify the targeted workloads that match `Restic`'s selector.
-4. Then, Stash operator inject a sidecar container named `stash` and mount the target volume in it.
+1. At first, a user creates a `Secret`. This secret holds the credentials to access the backend where backed up data will be stored. It also holds a password (`RESTIC_PASSWORD`) that will be used to encrypt the backed up data.
+2. Then, the user creates a `Restic` crd which specifies the targeted workload for backup. It also specifies the backend information where the backed up data will be stored.
+3. Stash operator watches for `Restic` crd. Once, it found a `Restic` crd, it identifies the targeted workloads that match `Restic`'s selector.
+4. Then, Stash operator injects a sidecar container named `stash` and mounts the target volume into it.
 5. Finally, `stash` sidecar container takes periodic backup of the volume to specified backend. It also creates a `Repository` crd in first backup which represents the original repository in the backend in Kubernetes native way.
 
 ## Backup
 
-In order to take backup, we need some sample data. Stash has some sample data in [stash-data](https://github.com/appscode/stash-data) repository. As [gitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo) volume has been deprecated, we will not use this repository as volume directly. Instead, we will create a [configMap](https://kubernetes.io/docs/concepts/storage/volumes/#configmap) from these data and use that ConfigMap as data source.
+In order to take back up, we need some sample data. Stash has some sample data in [stash-data](https://github.com/appscode/stash-data) repository. As [gitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo) volume has been deprecated, we are going to not use this repository as volume directly. Instead, we are going to create a [configMap](https://kubernetes.io/docs/concepts/storage/volumes/#configmap) from these data and use that ConfigMap as the data source.
```

Comment:
`we will not use` sounds better than `we are going to not use`. you can use `we are not going to use`.